test/extensions/filters/http/cache/cache_filter_utils_test.cc

```diff
@@ -0,0 +1,59 @@
+#include "extensions/filters/http/cache/cache_filter_utils.h"
+
+#include "test/test_common/utility.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+namespace {
+
+Http::TestRequestHeaderMapImpl non_cacheable_request_headers[] = {
```

Comment:
no static non-pod data

see https://github.com/envoyproxy/envoy/blob/master/STYLE.md

Just put these in the methods that test them.

Reply:
I declared them this way following the convention in `test/extensions/filters/http/cache/http_cache_utils_test.cc`. Should I change them to local variables or leave them?

Reply:
either local variables or class member variables. I think they should not have been that way in the original place either.